### PR TITLE
Bananas now have a 1% chance of spawning with the boomerang component

### PR DESCRIPTION
**Pull request by Rhials**
## About The Pull Request

Bananas now have a 1% chance to be bananarangs on initialization. The description is altered to make it slightly easier to notice, but otherwise there is no indication beyond throwing it.

https://user-images.githubusercontent.com/28870487/185721187-c849b86f-a246-48c5-a7ac-73121c581b48.mp4

## Why It's Good For The Game

bananarang hehe :)

Can be used for entertainment, bargaining, or engaging with a rarely seen component within the sandbox. 

## Changelog

:cl:
add: Bananas now have a 1% chance to be boomerang-bananas. These look identical to normal bananas, aside from a slightly unique examine text. Make sure to check your bananas before peeling it to slip the HOS, you may just have a rare item on your hands.
/:cl:

### CODE REVIEW

**Review by MrMelbert — CHANGES_REQUESTED** [review]

File: code/modules/hydroponics/grown/banana.dm [hunk 1]
```diff
@@ -28,6 +28,12 @@
 	juice_results = list(/datum/reagent/consumable/banana = 0)
 	distill_reagent = /datum/reagent/consumable/ethanol/bananahonk
 
+/obj/item/food/grown/banana/Initialize(mapload)
+	. = ..()
+	if(rand(100) == 1)
+		AddComponent(/datum/component/boomerang, throw_range+4, TRUE)
+		desc += " The curve one looks particularly acute."
```
[the remark this answers is not in the document]
Comment (Rhials) [reply code/modules/hydroponics/grown/banana.dm]: I swear to god I looked this over like 10 times before submitting how is there a typo

**Comment by IHateGeese** [comment]
This is the best request i have ever seen, merge now.

**Comment by drIcepop** [comment]
So when does the banana explode into a maxcap?

**Comment by Shadowflame909** [comment]
Clown Buff = fun things to work towards 

I love clown

File: code/modules/hydroponics/grown/banana.dm [hunk 2]
```diff
@@ -28,6 +28,12 @@
 	juice_results = list(/datum/reagent/consumable/banana = 0)
 	distill_reagent = /datum/reagent/consumable/ethanol/bananahonk
 
+/obj/item/food/grown/banana/Initialize(mapload)
+	. = ..()
+	if(prob(1))
+		AddComponent(/datum/component/boomerang, throw_range + 4, TRUE)
```
Comment (SuperNovaa41) [inline code/modules/hydroponics/grown/banana.dm:34]:
```suggestion
		AddComponent(/datum/component/boomerang, boomerang_throw_range = throw_range + 4, thrower_easy_catch_enabled = TRUE)
```

**Comment by Wallemations** [comment]
My favorite kind of PR
The hyper-specific rare random event with little game impact but big comedic value

**Comment by timothymtorres** [comment]
What happens if you have the squish or prickly trait?  Will the banana detonate on the mob that threw it?

**Comment by Wallemations** [comment]
> What happens if you have the squish or prickly trait?  Will the banana detonate on the mob that threw it?

If they do I think that's extremely funny

**Comment by Greenfire575** [comment]
merge right now

**Comment by SinfulBliss** [comment]
Do boomerang banana's peels retain this component? That would be pretty epic.

**Pull request merged**